spa/src/assets/index.scss

```diff
@@ -141,3 +141,72 @@ table {
     text-transform: uppercase;
 
 }
+
+X3DCanvas .x_ite-private-splash-screen {
```

Comment:
@smile0711 There is a cleaner way to write these selectors with SCSS. Instead of having `X3DCanvas <someSelector>` multiple times, you can do this:
```
X3DCanvas {
  .x_ite-private-splash-screen {
     ...
  }
  
  .x_ite-private-spinner {
    ...
  }
  
  .x_ite-private-progress {
    ...
  }
  
  .x_ite-private-x_ite {
    ...
  }
}
```
You can read about SCSS nesting [here](https://sass-lang.com/guide#topic-3).

Also, since this is just the index.scss, it's better to have rules pertaining to specific things in their own files (known as partials in SCSS parlance) and import them. So I would have created a file `spa/src/assets/_xite.scss`, and then in index.scss at the top I would `@import 'xite'`. There's a little blurb about it [here](https://sass-lang.com/guide#topic-4).

This looks good otherwise, so if you could please adopt these patterns, this PR is ready for merge.

Thanks!